src/init.cpp

```diff
@@ -1342,7 +1342,7 @@ bool AppInitMain(const util::Ref& context, NodeContext& node, interfaces::BlockA
     node.scheduler = MakeUnique<CScheduler>();
 
     // Start the lightweight task scheduler thread
-    threadGroup.create_thread([&] { TraceThread("scheduler", [&] { node.scheduler->serviceQueue(); }); });
+    g_scheduler = std::thread(&TraceThread<std::function<void()>>, "scheduler", [&] { node.scheduler->serviceQueue(); });
```

Comment:
I'm probably missing some context so feel free to ignore: can this continue to use a lambda rather than `std::function` for possibly less compile time / run time overhead? This compiles and the tests pass for me locally.
```diff
-    g_scheduler = std::thread(&TraceThread<std::function<void()>>, "scheduler", [&] { node.scheduler->serviceQueue(); });
+    g_scheduler = std::thread([&] { TraceThread("scheduler", [&] { node.scheduler->serviceQueue(); }); });
```

Reply:
@jonatack also #19064

Reply:
This change is now included.